### ISSUE TITLE
 Gun restrictions for each group.

### ISSUE THREAD

**Issue by tester00125** [uEssentials/uEssentials#238]
Gun restrictions allow players can usable some guns.

ex. 
Group A can usable Eaglefire only.
Group B can usable Peacemaker only.

I think this function will make a role-playing on unturned. players might be a grouping for survival.

**Comment by leonardosnt** [comment]
:+1:

**Comment by lammas112** [comment]
Yes would be good then some PvP servers can disable rocket launchers on their server.

**Comment by cellinho** [comment]
Disabling weapons can be done by deleting the assets file from unturned folder.
